src/item.cpp

```diff
@@ -3903,9 +3911,10 @@ units::volume item::volume( bool integral ) const
     return ret;
 }
 
-int item::lift_strength() const
+std::int64_t item::lift_strength() const
 {
-    return std::max( weight() / 10000_gram, 1 );
+    const std::int64_t mass = units::to_gram( weight() );
+    return std::max<std::int64_t>( mass / 10000, 1 );
```

Comment:
As you convert the value into the dimension of the strength (character attribute), which is an `int` itself, the function should continue to return `int`. We don't need the extended range of `int64_t`.

It is also used (in `player::can_lift`) in comparison with other `int`s.

Reply:
Same thing for gibbed weight then?

Reply:
Yes. As soon as you convert it into grams, you don't need the extended range of `int64` anymore.